guide/using-java.md

```diff
@@ -24,17 +24,17 @@ The Java Development Kit (JDK) can be downloaded for free from the [Sun
 Developer Network](http://www.oracle.com/technetwork/java/javase/downloads).
 Make sure you choose the most recent release and the Standard Edition (SE) of
 the JDK 8. For Windows, make also sure you have selected the 64 bit version
-since webots is incompatible with the 32 bit version. Then follow the
+since webots is not compatible with the 32 bit version. Then, follow the
```

Comment:
Another comment not related to your changes that would be convenient to fix in this PR: webots -> Webots